go.mod

```diff
@@ -44,7 +44,7 @@ require (
 	google.golang.org/grpc v1.19.1 // indirect
 	gopkg.in/check.v1 v1.0.0-20180628173108-788fd7840127 // indirect
 	gopkg.in/yaml.v2 v2.2.2 // indirect
-	k8s.io/api v0.0.0-20190226173710-145d52631d00 // indirect
+	k8s.io/api v0.0.0-20190226173710-145d52631d00
```

Comment:
do you know why this go converted into a direct dependency?

Reply:
Maybe now code contains `k8s.io/api/*` in `import` statements? :thinking: